config-linux.md

```diff
@@ -121,6 +121,9 @@ Each entry has the following structure:
 * **`uid`** *(uint32, OPTIONAL)* - id of device owner.
 * **`gid`** *(uint32, OPTIONAL)* - id of device group.
 
+If a `devices` contains duplicated devices with same `path`, the runtime MUST generate an error.
```

Comment:
Duplication in the `devices` array is a subset of this issue, which is that a path in `devices` already exists in the container filesystem being constructed.  Besides an earlier `devices` entry, this could also happen if a file (in the "everything is a file" sense) exists in the `root.path` directory at the target location, although you could also have conflicts sure to `mounts` entries.  I think this line should be under the `path` entry above with text like:

> If a [file][1] already exists at `path` that does not match the requested file, the runtime MUST generate an error instead of creating the requested file.

[1]: http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_164